x/bank/simulation/operations.go

```diff
@@ -82,13 +84,14 @@ func SimulateMsgSend(ak types.AccountKeeper, bk keeper.Keeper) simtypes.Operatio
 
 // SimulateMsgSendToModuleAccount tests and runs a single msg send where both
 // accounts already exist.
-func SimulateMsgSendToModuleAccount(ak types.AccountKeeper, bk keeper.Keeper) simtypes.Operation {
+func SimulateMsgSendToModuleAccount(ak types.AccountKeeper, bk keeper.Keeper, moduleAccCount int) simtypes.Operation {
 	return func(
 		r *rand.Rand, app *baseapp.BaseApp, ctx sdk.Context,
 		accs []simtypes.Account, chainID string,
 	) (simtypes.OperationMsg, []simtypes.FutureOperation, error) {
 		from := accs[0]
-		to := accs[1]
+
+		to := getModuleAccounts(ak, ctx, moduleAccCount)[0]
```

Comment:
@robert-zaremba  I have implemented getModuleAccounts to generate module accounts. i hope it make sense for you.